cmd/skipper/defaultfilterflags.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/zalando/skipper/eskip"
+)
+
+const (
+	defaultPrependFiltersUsage = "set of default filters to apply to prepend to all filters of all routes"
+	defaultAppendFiltersUsage  = "set of default filters to apply to append to all filters of all routes"
+)
+
+type defaultFiltersFlags struct {
+	filters []*eskip.Filter
+}
+
+func (dpf *defaultFiltersFlags) String() string {
+	res := make([]string, len(dpf.filters))
```

Comment:
shouldn't this be `make([]string, 0, len(dpf.filters))`?

Reply:
The append below will start appending elements after `len(dpf.filters)`

Reply:
:+1:

Reply:
it's not different if you have no `if ... continue` or `break`

Reply:
it should be also fine if we just remove this method

Reply:
my bad, must implement Stringer

Reply:
we decided to drop this and just return a static string instead

Reply:
@szuecs if it was only about the initialization, you'd be right. The comment is about how the append() will behave afterwards

Reply:
https://play.golang.org/p/GSEH2W1uANR @lmineiro you are right